### ISSUE TITLE
Future of katacoda interactive trainings?

### ISSUE THREAD

**Issue by marcosnils** [kubernetes/website#34007]
**Question**

Hi, I'm the the co-creator of [play-with-kubernetes ](https://play-with-k8s.com/) an *active *opens source** (https://github.com/play-with-docker) tool that allows users to rapidly create clusters and have a hands on experience to play with k8s features which is listed as an alternative to Katacoda in all the official k8s tutorials (https://kubernetes.io/docs/tasks/configure-pod-container/configure-service-account/#before-you-begin)

Katacoda has been acquired by O'Reilly some yearsago and I've recently discovered that they're planning to shut down all their public content and keep the platform for their internal usage only (https://www.oreilly.com/online-learning/leveraging-katacoda-technology.html). This is a bit  sad and surprising since all the community efforts in creating content and tutorials for katacoda will be usless any time soon. 

I've seen some issues like https://github.com/kubernetes/website/pull/34005 and https://github.com/kubernetes/website/issues/33918 cc @sftim @Sea-n which already mention this and I was also a bit surprised that PWK wasn't mentioned there. I've been supporting the platform for ~6 years and it currently has more than ~300k registeded users.

I'd like to see if there's any interest in talking about how we could leverage more on PWK to keep adding value to the k8s community. Happy to chat about future ideas and collaborations.

**Comment by sftim** [comment]
The best place to raise this is in the #sig-docs channel on Kubernetes Slack. Would you be willing to contribute to that discussion there @marcosnils? If you're not already in that Slack workspace, visit https://slack.k8s.io/ for an invitation.

/remove-kind support

**Comment by sftim** [comment]
The katacoda trainings remain in place past the shutdown of katacoda.com
/close

**Comment by marcosnils** [comment]
@sftim would it be proper to reopen given the latest announcements? https://kubernetes.io/blog/2023/02/14/kubernetes-katacoda-tutorials-stop-from-2023-03-31/

**Comment by sftim** [comment]
> @sftim would it be proper to reopen given the latest announcements? https://kubernetes.io/blog/2023/02/14/kubernetes-katacoda-tutorials-stop-from-2023-03-31/

No. Please use discussion https://github.com/kubernetes/website/discussions/38878 for discussions, or the existing tracking issue.
